### DISCUSSION TITLE
OVOS Workshop status

### DISCUSSION THREAD

**Discussion by wita-sec** [OpenVoiceOS/ovos-buildroot#227] [Q&A] [answered]
Is OVOS Workshop supposed to officially support Skill development, and will it be kept compatible from now on?
Also will there be documentation on this at some point in future?
As of now i would go ahead with my skill migration from Mycroft to OVOS based on what i can see in the OVOS Workshop code.

**Comment by builderjer** [comment]
Try this to help with the skill migration 
https://github.com/mikejgray/ovos-skill-projen

**Comment by JarbasAl** [comment, marked as answer]
ovos-workshop is the main library to create skills and voice apps, it is not going away anytime soon! probably one of the most important ovos components

developer docs are here https://openvoiceos.github.io/ovos-technical-manual , the examples for skills will always import from ovos-workshop

**Comment by wita-sec** [comment]
Thank you for your answers.

**Reply by JarbasAl** [reply]
anytime, feel free to open more questions!

trying to keep 1 per thread for easy QA search for new users
